Makefile.ibm

```diff
@@ -77,12 +78,26 @@ docker-login:
 	@echo $(DOCKER_PASS_ART) | docker login -u $(DOCKER_USER_ART) --password-stdin $(DOCKER_REGISTRY_ART);
 	@echo $(DOCKER_PASS_ICR) | docker login -u $(DOCKER_USER_ICR) --password-stdin $(DOCKER_REGISTRY_ICR);
 
-docker-publish-images: docker-login
-	for image_name in $(DOCKER_IMAGES_TAGGED) ; do                                             \
-		for registry in $(DOCKER_REGISTRIES) ; do                                         	   \
-			$(MAKE) docker-publish-image                                              		   \
-			IMAGE_NAME=$${image_name} DOCKER_REGISTRY=$${registry}; 						   \
-		done                                                                              	   \
+docker-publish-images:
+	# Tagged UBI images in special way (since they are pre-tagged); these will be the frozen versions for UBI images
+	if [ -n "$(TRAVIS_TAG)" ]; then \
+		docker tag $(DOCKER_DOMAIN_LOCAL)/detect-secrets:redhat-ubi $(DOCKER_DOMAIN_LOCAL)/detect-secrets:$(subst +,.,$(TRAVIS_TAG))-redhat-ubi ;  \
+		docker tag $(DOCKER_DOMAIN_LOCAL)/detect-secrets:redhat-ubi-custom $(DOCKER_DOMAIN_LOCAL)/detect-secrets:$(subst +,.,$(TRAVIS_TAG))-redhat-ubi-custom ;  \
+	fi
```

Comment:
I may have missed something, but isn't this block very similar to https://github.com/IBM/detect-secrets/blob/d31fe3d740f4c3abfa5a230c0a888ea633b1f703/Makefile.ibm#L139-L140?

My question is, why it necessary to have this block here as well?

Reply:
There are two if TRAVIS_TAG  statements .... one inside deploy target and another inside docker-publish-images target:

Reply:
Gotcha, I'm good with your approach. Thanks for the explanation!